### PR TITLE
refactor(all): update antd components and remove deprecated props

### PR DESCRIPTION
**Pull request by greatertomi**
# Checklist

- [x] JIRA - LW-7409
- [ ] Proper tests implemented
- [ ] Screenshots added.

---

## Proposed solution

This PR updates the props marked deprecated in components such as `Dropdown` and `Popup` to align with `antd` version 4.20+.

## Testing

Regression testing should be carried out on occurrences of Dropdown, Menu, Select, Autocomplete, Tooltip and Modal/Popup/Drawer.

## Screenshots

Attach screenshots here if implementation involves some UI changes

---
# Allure report
`allure-report-publisher` generated test report!

**smokeTests**: ❌ [test report](https://lace-qa:8PP5wBaDV6UoXj@dq4ajm5i5q7bz.cloudfront.net/linux/chrome/1645/5950614827/index.html) for [22e05034](https://github.com/input-output-hk/lace/pull/409/commits/22e050346afb50f61f7db8212ce5e3c4fd0c74f7)
|       | passed | failed | skipped | flaky | total | result |
|-------|--------|--------|---------|-------|-------|--------|
| Total | 30     | 5      | 0       | 0     | 35    | ❌     |

### CODE REVIEW

File: apps/browser-extension-wallet/src/components/MainMenu/UserMenu/components/UserMenu.module.scss [hunk 1]
```diff
@@ -154,7 +162,7 @@
   }
 }
 
-.networkChoise {
+.networkSwitcher {
```
Comment (VanessaPC) [inline apps/browser-extension-wallet/src/components/MainMenu/UserMenu/components/UserMenu.module.scss:165]: `networkSwitch`,  `switcher` doesn't exit.

File: packages/e2e-tests/src/elements/menuHeader.ts [hunk 2]
```diff
@@ -6,7 +6,7 @@ export class MenuHeader {
   private AVATAR_ON_MENU = '[data-testid="header-menu"] [data-testid="user-avatar"]';
   private CHEVRON_DOWN = '[data-testid="chevron-down"]';
   private CHEVRON_UP = '[data-testid="chevron-up"]';
-  private CONTAINER = '//ul[@data-testid="header-menu"]';
+  private CONTAINER = '//ul[@data-menu-list="true"]';
```
Comment (VanessaPC) [inline packages/e2e-tests/src/elements/menuHeader.ts:9]: why is it changed from `data-testid`?

Comment (shawnbusuttil) [reply packages/e2e-tests/src/elements/menuHeader.ts:9]: The dropdown won't recognize the data testid attribute.

File: apps/browser-extension-wallet/src/components/MainMenu/UserMenu/components/UserMenu.module.scss (hunk 1 above)
Comment (shawnbusuttil) [reply apps/browser-extension-wallet/src/components/MainMenu/UserMenu/components/UserMenu.module.scss:165]: Wdym?

**Comment by shawnbusuttil** [comment]
Smoke test passing. WIP tag can be removed.

File: apps/browser-extension-wallet/src/components/MainMenu/UserMenu/components/UserMenu.module.scss (hunk 1 above)
Comment (VanessaPC) [reply apps/browser-extension-wallet/src/components/MainMenu/UserMenu/components/UserMenu.module.scss:165]: NVM.

**Review by VanessaPC — APPROVED** [review]

**Review by gabriela-ponce — APPROVED** [review]

**Review by rhyslbw — APPROVED** [review]

**Review by shawnbusuttil — APPROVED** [review]

File: apps/browser-extension-wallet/src/components/MainMenu/UserMenu/components/NetworkSwitcher.tsx [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+export const NetworkSwitcher = ({ onClick }: NetworkSwitcherProps): React.ReactElement => {
+  const { t } = useTranslation();
+  const { environmentName } = useWalletStore();
+
+  return (
+    <div
+      data-testid="header-menu-network-switcher-container"
+      className={cn(styles.menuItem, styles.cta)}
+      onClick={() => onClick()}
```
Comment (mrcnk) [inline apps/browser-extension-wallet/src/components/MainMenu/UserMenu/components/NetworkSwitcher.tsx:19]:
```suggestion
      onClick={onClick}
```

**Review by mrcnk — APPROVED** [review]

**Pull request closed without merging**